lib/phut/syntax.rb

```diff
@@ -1,4 +1,4 @@
-require 'phut/phost'
+require 'phut/vhost'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.